### DISCUSSION TITLE
Wherefore no release?

### DISCUSSION THREAD

**Discussion by dwildstr** [hassio-addons/addon-grocy#456] [General]
As the first birthday of the implemented version of Grocy in the most recent addon release is rolling around, I'm wondering when the current codebase will be turned into a release. The current codebase seems to be a fully-functioning containerization of Grocy 4.2.0, which implements a few new features and many bugfixes. Users of Grocy under HASSIO have occasionally reported bugs from time to time which are in fact fixed in Grocy itself (e.g. grocy/grocy#2510, grocy/grocy#2514)

Is there more testing required on the addon implementation, or a new release of grocy expected soon (the development version of grocy isn't that far changed from 4.2.0, so this would be a surprise), or a need to merge #422 or some other good reason for delay? It seems that since addons are chiefly configuration rather than code, that the delay from a new release of the underlying project to a release of an addon wrapper for that project should be able to be pretty short under most circumstances.

**Comment by valadas** [comment]
I would really welcome a new release, a lot of bugfixes available and just needs a release here. I am happy to test or help in any way I can.
